### ISSUE TITLE
"datetime" fields don't work in Strapi 4.11.4

### ISSUE THREAD

**Issue by web-admin** [strapi/strapi#17348]
### Required System information
Node.js version: 16
NPM version: 9
Strapi version: 4.11.4
Database: postgreSQL
Is your project Javascript or Typescript: typescript

### Describe the bug
All "datetime" fields change chosen date to 2 days before. Example: I've chosen 31.06.2023 -> I get 30.06.2023 -> and 29.06.2023 when saved an entry.

The issue occurs in PUBLISHER plugin too.

### Steps to reproduce the behavior

1. Go to any entry with "datetime" fields or PUBLISHER plugin active.
2. Click on field and chose any date -> after that the input will show the date 1 day before.
3. Save an entry and the date will go back 1 day again so finally you get 2 days before from the initially chosen date.

### Expected behavior
Value should not change when chosen/saved.

### Additional context
Rollback to Strapi 4.11.3 solved the issue.

**Comment by web-admin** [comment]
> > This is a templated message
> 
> Hello @web-admin,
> 
> We ask that you please follow the [issue template](https://raw.githubusercontent.com/strapi/strapi/master/.github/ISSUE_TEMPLATE/BUG_REPORT.md). A proper issue submission let's us better understand the origin of your bug and therefore help you. We will reopen your issue when we receive the issue following the template guidelines and properly fill out the template. You can see the template guidelines for bug reports [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
> 
> Please update the issue with the template and we can reopen this report.
> 
> Thank you.

Updated.

**Comment by joshuaellis** [comment]
You're missing `Steps to reproduce the behavior` :)

**Comment by web-admin** [comment]
> You're missing `Steps to reproduce the behavior` :)

Done.

**Comment by joshuaellis** [comment]
Does it still occur without the plugin? we don't maintain that and I want to be sure this is not interfering with anything.

**Comment by web-admin** [comment]
The problem occurred when we change the Strapi version from 4.11.3 to 4.11.4. After rollback everything back to normal. PUBLISHER is in the newest 1.3.7 version.

**Comment by joshuaellis** [comment]
> The problem occurred when we change the Strapi version from 4.11.3 to 4.11.4. After rollback everything back to normal. PUBLISHER is in the newest 1.3.7 version.

Yes, but just to confirm if you uninstall the plugin – the issue persists?

**Comment by web-admin** [comment]
Yup, the issue still persist without PUBLISHER.

**Comment by cangomescisco** [comment]
+1

**Comment by derrickmehaffy** [comment]
@joshuaellis you won't be able to reproduce this as your based in GMT if I remember right? This will only happen to anyone with a GMT offset and would generally only happen to date/times set as a specific timeframe where the GMT offset would either be a day in the past or future. It has to do with node's `new Date()` reading and not properly taking the date as a literal GMT but doing a server side conversion to GMT.

**Comment by joshuaellis** [comment]
> @joshuaellis you won't be able to reproduce this as your based in GMT if I remember right? This will only happen to anyone with a GMT offset and would generally only happen to date/times set as a specific timeframe where the GMT offset would either be a day in the past or future. It has to do with node's `new Date()` reading and not properly taking the date as a literal GMT but doing a server side conversion to GMT.

Thanks, yep GMT, i can change the location in browser though, so when we check this out i'll be sure to write those as notes.

**Comment by Freeko13** [comment]
+1

**Comment by rktyt** [comment]
Probably the same as #17210

**Comment by derrickmehaffy** [comment]
Raising sev as also raised by an EE customer per our internal escalation policy

**Comment by derrickmehaffy** [comment]
Internal TID 4451

**Comment by marob** [comment]
Should be fixed by https://github.com/strapi/design-system/pull/1241
